tests/py/unit/test_tools_build_defect.py

```diff
@@ -0,0 +1,51 @@
+from mat3ra.made.material import Material
+from mat3ra.made.tools.build.defect import PointDefectBuilderParameters, PointDefectConfiguration, create_defect
+
+clean_material = Material.create(Material.default_config)
+
+
+def test_create_vacancy():
+    # vacancy in place of 0 element
+    configuration = PointDefectConfiguration(crystal=clean_material, defect_type="vacancy", site_id=0)
+    defect = create_defect(configuration)
+
+    assert len(defect.basis["elements"]) == 1
+
+
+def test_create_substitution():
+    # Substitution of Ge in place of Si at default site_id=0
+    configuration = PointDefectConfiguration(crystal=clean_material, defect_type="substitution", chemical_element="Ge")
+    defect = create_defect(configuration)
+
+    assert defect.basis["elements"] == [{"id": 0, "value": "Ge"}, {"id": 1, "value": "Si0+"}]
```

Comment:
What is this `Si0+`?? The elements should have actual element names only without anything else